scripts/src/java/org/oppia/android/scripts/common/GitHubClient.kt

```diff
@@ -68,13 +68,14 @@ class GitHubClient(
       val call = gitHubService.fetchOpenIssues(repoOwner, repoName, authorizationBearer, pageNumber)
       // Deferred blocking I/O operation to the dedicated I/O dispatcher.
       val response = withContext(Dispatchers.IO) { call.execute() }
+
       check(response.isSuccessful()) {
         "Failed to fetch issues at page $pageNumber: ${response.code()}\n${call.request()}" +
           "\n${response.errorBody()}."
       }
       return@async checkNotNull(response.body()) {
         "No issues response from GitHub for page: $pageNumber."
-      }
+      }.filter { it.pullRequest == null }
```

Comment:
What does this mean?

Reply:
The line .filter { it.pullRequest == null } is applying a filter to the list of issues returned in the response.body() so that only issues that are not pull requests are included in the final list.

Reply:
Sounds good.